test/Makefile

```diff
@@ -209,3 +211,5 @@ $(RESULTS_DIR)/d_do_test$(EXE): d_do_test.d $(RESULTS_DIR)/.created
 	$(QUIET)$(DMD) -conf= $(MODEL_FLAG) -unittest -run d_do_test.d -unittest
 	$(QUIET)$(DMD) -conf= $(MODEL_FLAG) -od$(RESULTS_DIR) -of$(RESULTS_DIR)$(DSEP)d_do_test$(EXE) d_do_test.d
 
+test_dub: $(DMD)
+	cd ./dub_package && $(DUB) test --compiler=$(abspath $(DMD)) --force
```

Comment:
Also I decided to avoid overwriting the DFLAGS as the `-fPIC` by default is lost then.